### PR TITLE
Hw2

### PR DESCRIPTION
**Pull request by khoanguyen0812**
@diegodulanto, @njie-DP. Sorry for posting this review up for you guys so late.

### CODE REVIEW

File: HW2/Exploring gapminder and use dplyr.Rmd [hunk 1]
```diff
@@ -39,13 +39,19 @@ gapminder %>%
```
Comment (diegodulanto) [inline HW2/Exploring gapminder and use dplyr.Rmd:39]: I like your countries and your code! Great job!

File: Participation/Mutating and summarizing data set.R [hunk 2]
```diff
@@ -48,3 +48,15 @@ bfiscore %>%
                 min = ~ min(.x, na.rm = TRUE),
                 max = ~ max(.x, na.rm = TRUE))))
```
Comment (diegodulanto) [inline Participation/Mutating and summarizing data set.R:50]: Nice job keeping your code clean by entering a new line!

**Review by diegodulanto — COMMENTED** [review]
Your coding looks great!

File: HW2/Exploring gapminder and use dplyr.Rmd [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+```
+
+## 1.3
+
+Make a new variable in gapminder for the change in life expectancy from the previous measurement. Filter this table to show all of the entries that have experienced a drop in life expectancy.
+
+
+```{r}
+gapminder %>% 
+  mutate(dropLifeExp <- lag(lifeExp))
```
Comment (njie-DP) [inline HW2/Exploring gapminder and use dplyr.Rmd:47]: I think you should also filter to show the entries with a drop in life expectancy.

**Review by njie-DP — COMMENTED** [review]
Your codes look really clean!

**Pull request merged**